### PR TITLE
Add students for 2018 season.

### PR DESCRIPTION
**Pull request by dpshelio**
List of all the students for this year

- [x] @MananAgarwal
- [x] @sushobhana
- [x] @rkiman
- [x] @ritiek
- [x] @nikita-astronaut
- [x] @abotiamnot
- [x] @swapsha96
- [x] @prakharcode
- [x] @vn-ki

Make a PR to this branch (`gsoc2018`) to add or fix your rss feed.

### CODE REVIEW

**Comment by nikita-astronaut** [comment]
Hello! Can you please tell what I should do about this?:)-- Никита Астраханцев20:46, 28 April 2018, "David Pérez-Suárez" <notifications@github.com>:List of all the students for this year

 @MananAgarwal
 @sushobhana
 @rkiman
 @ritiek
 @nikita-astronaut
 @abotiamnot
 @swapsha96
 @prakharcode
 @vn-ki

Make a PR to this branch (gsoc2018) to add or fix your rss feed.



You can view, comment on, or merge this pull request online at:
  https://github.com/OpenAstronomy/Universe_OA/pull/8

Commit Summary

  Add students for 2018 season.


File Changes

  
    M
    gsoc.yml
    (28)
  
  
    M
    gsoc_times.yml
    (10)
  


Patch Links:

  https://github.com/OpenAstronomy/Universe_OA/pull/8.patch
  https://github.com/OpenAstronomy/Universe_OA/pull/8.diff

**Comment by abotiamnot** [comment]
@nikita-astronaut Hello Nikita! We have to prepare a RSS feed for our blog posts for OpenAstronomy! Like you can check out Sushobhana's feed!

**Comment by dpshelio** [comment]
I've just sent the email explaining. But yeah, @abotiamnot is right!

**Review by dpshelio — COMMENTED** [review]
We are almost there!

File: gsoc.yml [hunk 1]
```diff
@@ -1,3 +1,31 @@
+gsoc2018:
+  MananAgarwal:
+    rss_feed: "https://medium.com/feed/@manan_agarwal"
+    project: 'Astropy'
+  sushobhana:
+    rss_feed:  "https://medium.com/feed/@sushobhanapatra"
+    project: 'Astropy'
+  rkiman:
+    rss_feed: "https://rociokiman.blogspot.com/feeds/posts/default?alt=rss" # set up tag? FIXME?
```
Comment (dpshelio) [inline gsoc.yml]: @rkiman could you update the feed so only gets your GSoC posts?
Read about [how to do so] (https://fedoraproject.org/wiki/How_to_provide_a_category_specific_feed_using_Blogspot_or_Blogger?rd=How_to_provide_a_category_specific_feed_using_Blogger).

File: gsoc.yml [hunk 2]
```diff
@@ -1,3 +1,31 @@
...
+    project: 'Astropy'
+  ritiek:
+    rss_feed: "https://ritiek.github.io/feed.xml"  # set up tag? FIXME?
```
Comment (dpshelio) [inline gsoc.yml]: @ritiek Would be possible to get a feed for only the posts you would do related with GSoC?

**Comment by prakharcode** [comment]
I dont see a tick, any issue? :sweat_smile:

**Pull request merged**